Documentation/Contributing/rook-test-framework.md

```diff
@@ -9,7 +9,7 @@ The framework includes scripts for starting Kubernetes so users can
 quickly spin up a Kubernetes cluster. The tests are generally designed to install Rook, run tests, and uninstall Rook.
 
 The CI runs the integration tests with each PR and each master or release branch build.
-If the tests fail in a PR, a tmate session is started which will allow you to connect via ssh and troubleshoot the failure.
+If the tests fail in a PR, restart CI with debug mode from github UI or add label `debug-ci`  and push your changes again and a tmate session will be available for troubleshooting for a sometime and a tmate session is started which will allow you to connect via ssh and troubleshoot the failure.
```

Comment:
Instead of duplicating the content, let's just link to the tmate section in the other doc.